sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SymbolicExecution/ProgramState.cs

```diff
@@ -65,6 +78,11 @@ internal ProgramState()
         {
         }
 
+        private static ImmutableList<T> BuildImmutableList<T>(params T[] items)

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private method 'BuildImmutableList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1144)